portlist/portlist_linux_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build linux
```

Comment:
this is redundant with the filename, but also fine if you like it

Reply:
Oh, didn't know that Go groks `_<platform>_test.go` suffixes. Removed.